spec/request/organizations_spec.rb

```diff
@@ -1514,6 +1540,33 @@ module VCAP::CloudController
           end
         end
 
+        context 'by partial_usernames and origins' do
+          before do
+            allow(uaa_client).to receive(:users_for_ids).with([org_manager.guid]).and_return({
+              org_manager.guid => { 'username' => 'b-mcjam', 'origin' => 'Okta' },
```

Comment:
I know it's a mock, but does the result really include the partial username (`b-mcjam`), not the full username (`rob-mcjam`)?

Reply:
Thanks, the mock is supposed to have `rob-mcjam` as username